docs/datasets/index.rst

```diff
@@ -52,10 +52,83 @@ Note that in Gammapy, 2D image analyses are done with 3D cubes with a single
 energy bin, e.g. for modeling and fitting,
 see the `2D map analysis tutorial <./tutorials/image_analysis.html>`__.
 
+To analyse multiple runs, you can either stack the datasets together, or perform
+a joint fit across multiple datasets.
 
-Getting Started
-===============
+Stacking Multiple Datasets
+==========================
 
+Stacking datasets implies that the counts, background and reduced IRFs from all the
```

Comment:
One could put some motivation on why stacking is needed at all: reduction of computation effort.